src/components/RickAPI/RickAPI.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect, useCallback } from "react";
+import Button from "@mui/material/Button";
+
+type resultProps = {
+  name: string;
+  gender: string;
+  species: string;
+  image: string;
+};
+
+export const RickAPI = () => {
```

Comment:
You should probably use camelCase here

Reply:
Done. I changed it to the "RickData" name.

Reply:
This is a minor issue, but to be precise - camelCase starts from a lowercase character (rickApi).

Reply:
OK, I know this rule, but it doesn't work in this case. Errors pop up - I tried. I think the reason is that it's the component name.